api.md

```diff
@@ -520,9 +521,13 @@ are before the cut-off point, but only one chunk's worth.
 
 #### Required Arguments [](drop_chunks-required-arguments)
 
+Function requires at least one of the following timestamp arguments. When both timestamp arguments
+are used, the function returns the intersection of those two timestamp ranges.
```

Comment:
I would also specify that the arguments have same semantics a show_chunks.

Reply:
Done